### ISSUE TITLE
Copy> Paste: with Structural Editing ON and you highlight Unbalanced Form to copy

### ISSUE THREAD

**Issue by rsachdeva** [cursiveclojure/cursive#807]
To explain suppose we have this code-
```
(do 
   (function1)
   (function2)
   (function3) )
```

And I highlight-
```
(do 
   (function1)
   (function2)
   (function3)
```

The last bracket is missing.
Now I do copy>paste.
When this is allowed in Structural Editing, it results in Unbalanced code in the Editor. And to really get it balanced you have to turn off Structural editing.

Can we disable so that if the highlighted code is unbalanced the Copy>Paste does not happen when Structural Editing is ON?  Not sure if this is possible. Otherwise whats the alternative best way in Cursive to do copy>paste multiple functions in a balanced way?

**Comment by cursiveclojure** [comment]
Yes, I've been thinking about this, see: https://twitter.com/hlship/status/555387290642432000

**Comment by rsachdeva** [comment]
Glad to see that others have had similar experience! Hope we can get to smarter copy>paste! :+1:

**Comment by rsachdeva** [comment]
Any plans for this support?
